### PR TITLE
Update bug bounty with v3 contracts

### PR DESCRIPTION
**Pull request by abandeali1**

### CODE REVIEW

**Review by hysz — APPROVED** [review]

File: mdx/guides/bug-bounty-program.mdx [hunk 1]
```diff
@@ -18,22 +18,43 @@ The following are examples of types of vulnerabilities that are of interest:
...
 # Scope
 
 The following contracts are in scope of the bug bounty. Please note that any bugs already reported are considered out of scope.
 
+## 3.0
+
+3.0 submissions should be based off of the contracts as of commit [fb8360edfd4f42f2d2b127b95c156eb1b0daa02b](https://github.com/0xProject/0x-monorepo/tree/fb8360edfd4f42f2d2b127b95c156eb1b0daa02b/contracts).
+The following contracts are considered within scope of this bug bounty:
+
+-   [asset-proxy/contracts/src/ERC20BridgeProxy.sol](https://github.com/0xProject/0x-monorepo/blob/fb8360edfd4f42f2d2b127b95c156eb1b0daa02b/contracts/asset-proxy/contracts/src/ERC20BridgeProxy.sol) ([specification](https://github.com/0xProject/0x-protocol-specification/blob/master/asset-proxy/erc20-bridge-proxy.md))
```
Comment (jalextowle) [inline mdx/guides/bug-bounty-program.mdx:38]: Nit: Should this contract be included in both the `3.0` and its own bounty?

**Review by jalextowle — APPROVED** [review]
Looks good aside from a nit!

File: mdx/guides/bug-bounty-program.mdx [hunk 2]
```diff
@@ -18,22 +18,43 @@ The following are examples of types of vulnerabilities that are of interest:
...
     -   Any assets get unexpectedly "stuck" in a contract with regular use of the contract's public methods.
+    -   An action taken in the staking contracts is applied to an incorrect epoch.
 -   Bypassing time locks
-    -   The `AssetProxyOwner` is allowed to bypass the timelock for transactions where it is not explicitly allowed to do so.
-    -   A user is allowed to bypass the `AssetProxyOwner`.
+    -   The `ZeroExGovernor` is allowed to bypass the timelock for transactions where it is not explicitly allowed to do so.
+    -   A user is allowed to bypass the `ZeroExGovernor`.
+-   Incorrect math calculations
+    -   Overflows or underflow result in unexpected behavior.
+    -   The staking reward payouts are incorrect.
```
Comment (dorothy-zbornak) [inline mdx/guides/bug-bounty-program.mdx:27]: Maybe this, to weed out submissions that are just slight imprecisions?
```suggestion
    -   The staking reward payouts are (significantly) incorrect.
```

Comment (jalextowle) [reply mdx/guides/bug-bounty-program.mdx:27]: Is there a threshold that you would consider significant? It seems like something like 10% would weed out "dust" imprecisions, but could also be too stringent.

Comment (dorothy-zbornak) [reply mdx/guides/bug-bounty-program.mdx:27]: Not sure if % is the best way to express this (% of what?). From the tests, calculations with an absolute error greater than `1e-12` would surprise me, but that kind of error might not be high wreak any real havoc. Maybe it's best to just leave this subjective.

Comment (abandeali1) [reply mdx/guides/bug-bounty-program.mdx:27]: Maybe just "excluding minor rounding errors"? I'd want to know about any small imprecisions that we aren't already aware of. Keep in mind that the bug bounty also pays out based on severity, so no big deal if there are some minor submissions.

File: mdx/guides/bug-bounty-program.mdx (hunk 1 above)
Comment (abandeali1) [reply mdx/guides/bug-bounty-program.mdx:38]: I decided to include it in both just to highlight that it is a new contract.

File: mdx/guides/bug-bounty-program.mdx (hunk 2 above)
Comment (abandeali1) [reply mdx/guides/bug-bounty-program.mdx:27]: Actually, we should also keep in mind that these are just examples of things that can qualify for the bounty. Probably best to leave them more open ended with that in mind.

Comment (dorothy-zbornak) [reply mdx/guides/bug-bounty-program.mdx:27]: It isn't a "rounding" error, though, if, for example, our logarithmic approximation is off.

**Review by dorothy-zbornak — APPROVED** [review]
:eyes: :+1:

**Pull request merged**